src/routes/cfp/+page.svelte

```diff
@@ -176,156 +193,268 @@
 <div class="page">
   <Window title={$t("cfp--title")}>
     <form on:submit|preventDefault={handleSubmit}>
-      <h4><Localized id="cfp--section-main" /></h4>
+      <!-- ENTRY START -->
```

Comment:
We aren't using these kind of comments.
If we really want to make a code section, we can use functions or modules.